### DISCUSSION TITLE
"For You" Seems a Bit Creepy...

### DISCUSSION THREAD

**Discussion by UxuginPython** [community/community#17069] [Other Feature Feedback, Questions, & Ideas]
It seems a lot like how Google and Facebook track your interests. I'm very concerned about privacy, especially since GitHub is now owned by Microsoft. In addition, I could not locate the source code for this feature. If the source code is available, please provide a link.

**Comment by jfullstackdev** [comment]
it's the nature of things, I think so, we are always left with limited choices

https://www.security.org/resources/data-tech-companies-have/
